packages/pipeline/src/ormconfig.ts

```diff
@@ -0,0 +1,42 @@
+import { ConnectionOptions } from 'typeorm';
```

Comment:
All file names should be snake-case: `orm_config.ts`

Reply:
Can't change this one as the TypeORM CLI requires a file with a specific name for configuration.